test/Microsoft.Azure.SignalR.E2ETests/Microsoft.Azure.SignalR.E2ETests.csproj

```diff
@@ -25,4 +25,9 @@
     <ProjectReference Include="..\..\src\Microsoft.Azure.SignalR.Management\Microsoft.Azure.SignalR.Management.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.Azure.SignalR.Common\Microsoft.Azure.SignalR.Common.csproj" />
   </ItemGroup>
+
+  <ItemGroup Condition=" '$(AzureSignalRSDKE2ETest)' == 'true' ">
```

Comment:
What is this for? It already have `ProjectReference` for Management above

Reply:
Only for nightly e2e test. Do you remember we want to e2e test including restore packages from myget?

Reply:
Then remove the one from above for this condition?